pkg/kube/kubehttp.go

```diff
@@ -34,6 +36,8 @@ func CreateScheme() (*apiruntime.Scheme, error) {
 		rbacv1.AddToScheme,
 		authv1.AddToScheme,
 		notificationv2.AddToScheme,
+		reflectorv1.AddToScheme,
```

Comment:
@luizbafilho it might be useful to just raise a quick ADR to record that we are not creating its own endpoints because XWZ